integrations/va-auth0/src/index.tsx

```diff
@@ -330,9 +407,22 @@ export default createIntegration({
         const url = new URL(`${issuerBaseUrl}/authorize`);
         url.searchParams.append('client_id', clientId);
         url.searchParams.append('response_type', 'code');
+        url.searchParams.append('scope', 'openid');
```

Comment:
Is it only when we want to include the claims?

Is it going to impact existing users?

Reply:
Yes forgot to add the conditional inclusion sorry. Adding it. For Auth0 by default there is no request for consent by the user at login for only the `openid` scope but it is best to have this conditionally request that scope in case some customer setup have this require consent enabled.